hail/python/test/hail/expr/test_expr.py

```diff
@@ -274,6 +274,10 @@ def test_rbind_multiple(self):
     def test_rbind_placement(self):
         self.assertEqual(hl.eval(5 / hl.rbind(5, lambda x: x)), 1.0)
 
+    def test_translate(self):
+        strs = [None, '', 'TATAN']
+        assert hl.eval(hl.literal(strs, 'array<str>').map(lambda x: x.translate({'T': 'A', 'A': 'T'}))) == [None, '', 'ATATN']
+
```

Comment:
Please test the error case as well, where the mapping contains strings longer than one character.